### PR TITLE
[ADD] .github: Add standard folder .github to add github templates

### PR DESCRIPTION
**Pull request by moylop260**
- Auto fill a template format for issues and pull request of github
- Add a contributing link in issues.

Example:
guidelines

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Please read here the [contribute doc](https://github.com/Vauxoo/pylint/blob/master/doc/contribute.rst)
```
Comment (PCManticore) [inline .github/CONTRIBUTING.md]: Please change this to our link.

Comment (moylop260) [inline .github/CONTRIBUTING.md]: hahaha copy&paste error sorry

Comment (moylop260) [inline .github/CONTRIBUTING.md]: Thanks for feedback fixed!

**Comment by PCManticore** [comment]
Thank you.

**Pull request merged**